### ISSUE TITLE
DEV | Eleven labs | Highlighted lessons should be child of Eleven Labs

### ISSUE THREAD

**Issue by usmaann1** [JuiceMind/JuiceMind#916]
![image](https://github.com/user-attachments/assets/92da6174-4fc2-47c3-a00f-b6ef7934aadf)

**Comment by juicemind-dev** [comment]
Yes, need to check with curriculum developers why they even have a folder here. Should just remove that folder.
